programs/do.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Thi script enables the building of Libra Move programs using the Libra Move IR compiler
+# built from the Solana Libra Github repo.
+#
+# Libra Move program source files are text files with the `.mvir` extension.  This script
+# compiles these files into the json representations of the libra `transaction::Program` data
+# structure which can then be loaded into on-chain program accounts.
+#
+# Dependencies:
+#   - The Solana Libra repo must be cloned and the rust development environment setup
+#     - `$ git clone https://github.com/solana-labs/libra.git`
```

Comment:
Do we want to suggest a tag?

Reply:
Doesn't really matter which one but I'll add one